x/bitcoin/keeper/keeper.go

```diff
@@ -202,18 +204,38 @@ func (k Keeper) GetTransactionFeeRate(ctx sdk.Context) sdk.Dec {
 	return result
 }
 
-// SetAddress stores the given address information
-func (k Keeper) SetAddress(ctx sdk.Context, address types.AddressInfo) {
-	k.getStore(ctx).Set(addrPrefix.Append(utils.LowerCaseKey(address.Address)), &address)
+// SetAddressInfo stores the given address information
+func (k Keeper) SetAddressInfo(ctx sdk.Context, address types.AddressInfo) {
+	key := depositAddrPrefix.Append(utils.LowerCaseKey(address.Address))
```

Comment:
It doesn't make sense to use something called `depositAddrPrefix` here. `SetAddressInfo` isn't just used for deposit addresses but also consolidation addresses.

Reply:
renamed to `addressInfoPrefix`